src/navigation/MainTabNavigator.tsx

```diff
@@ -108,18 +106,18 @@ export const MainTabNavigator = () => {
           tabRoutes?.inactiveBackgroundColor || tabNavigatorConfig.inactiveBackgroundColor,
         tabBarInactiveTintColor:
           tabRoutes?.inactiveTintColor || tabNavigatorConfig.inactiveTintColor,
-        tabBarItemStyle: { marginTop: normalize(0) },
         tabBarStyle: {
           backgroundColor: colors.surface,
           borderTopColor: colors.gray20,
-          borderTopWidth: 1,
-          height: tabBarHeight
+          borderTopWidth: 1
         },
+        tabBarItemStyle:
+          isPortrait && !hasSafeArea
+            ? { marginBottom: normalize(7), marginTop: normalize(4) }
+            : undefined,
```

Comment:
How about using the && operator instead of `undefined` here?

```
 tabBarItemStyle: isPortrait && !hasSafeArea && { marginBottom: normalize(7), marginTop: normalize(4) },
```

Reply:
because using only && would return false instead of undefined if it does not resolve to true